src/server/db/seed/index.ts

```diff
@@ -1,57 +1,64 @@
-const { PrismaClient } = require("@prisma/client");
-const { users, habits, checkIns, contacts } = require('./seedData.json');
+import { PrismaClient } from "@prisma/client";
+import { users, habits, routines} from './seedData.json';
 const prisma = new PrismaClient();
 
+interface HabitSeedData {
+    name: string
+    datesCompleted: string[]
+    checkInDay: string
+    ownerId: number
+}
+
+const habitsWithDateObjects = habits.map((habit: HabitSeedData) => {
+    return ({
+        ...habit,
+        datesCompleted: habit.datesCompleted.map((date: string) => new Date(date)) 
+    })
+})
+
+console.log("habitsWithDateObjects: ", habitsWithDateObjects);
+
 const load = async () => {
     try {
         await prisma.habit.deleteMany();
         console.log("Deleted records in habits table")
 
-        await prisma.checkIn.deleteMany();
-        console.log("Deleted records in checkIns table");
+        await prisma.routine.deleteMany();
```

Comment:
I think it's best practice to have the routine deleted first on re-seeding?. Just because Routines depend on the existence of Habits depend on the existence of Users. I could be off base here though and I'm fuzzy on whether this matters because of onDelete: Cascade

Reply:
I think your instinct is right. With the cascade setting, it shouldn't matter though, as long as we set it up right. If you seed the data a second time, does it work on your end? I can try that too.

Reply:
I tried it seeding again and it worked fine. It may not have worked without the cascading set up properly though.